code/operators/solution/operators.sol.cpp

```diff
@@ -6,88 +6,77 @@
 class Fraction {
  public:
   explicit Fraction(int i) : m_num(i), m_denom(1) {}
-  Fraction(int num, int denom) : m_num(num), m_denom(denom) {}
-
-  int num() const { return m_num; }
-  int denom() const { return m_denom; }
+  Fraction(int num, int denom) : m_num(num), m_denom(denom) {
+    normalize();
+  }
 
-  void normalize() {
-    const int gcd = std::gcd(m_num, m_denom);
-    m_num /= gcd;
-    m_denom /= gcd;
+  friend std::ostream& operator<<(std::ostream& os, Fraction const & f) {
+    os << f.m_num << "/" << f.m_denom;
+    return os;
   }
 
-  Fraction& operator*=(int i) {
+  Fraction & operator*=(int i) {
     m_num *= i;
     normalize();
     return *this;
   }
 
-  Fraction& operator*=(Fraction f) {
-    m_num *= f.num();
-    m_denom *= f.denom();
+  Fraction & operator*=(Fraction const & o) {
```

Comment:
It hurts my soul to see you trample my beautiful code with these unnecessary whitespaces and use of east-const :)

Should we add a `.clang-format` at some point to format the examples? Since clang-format-14 it can also handle east/west const.

Reply:
east const is the only logical way to use const keyword. west const should just be banned in my opinion.
But yes, I agree, we should have a .clang-format at some point. Not the most urgent though.